templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
```

Comment:
Pat and I discovered yesterday that you don't need to quote assignment, ever, in any of the popular shells.